src/hotspot/share/opto/postaloc.cpp

```diff
@@ -253,12 +254,14 @@ int PhaseChaitin::elide_copy( Node *n, int k, Block *current_block, Node_List &v
 
   // Phis and 2-address instructions cannot change registers so easily - their
   // outputs must match their input.
-  if( !can_change_regs )
+  if (!can_change_regs) {
     return blk_adjust;          // Only check stupid copies!
-
+  }
```

Comment:
Style nit here and in other places: weren't these blocks separated by newline before?

Reply:
I'm not sure what you mean. Can you make these Suggestions? Thanks.

Reply:
> weren't these blocks separated by newline before?

They were, but also there were no braces around.

Reply:
👍